lgc/util/PassManager.cpp

```diff
@@ -162,8 +188,17 @@ lgc::LegacyPassManager *lgc::LegacyPassManager::Create() {
 // Create a PassManagerImpl
 //
 // @param lgcContext : LgcContext to get TargetMachine and LLVMContext from
-lgc::PassManager *lgc::PassManager::Create(LgcContext *lgcContext) {
-  return new PassManagerImpl(lgcContext->getTargetMachine(), lgcContext->getContext());
+std::unique_ptr<lgc::PassManager> lgc::PassManager::Create(LgcContext *lgcContext) {
+  return std::unique_ptr<lgc::PassManager>(
+      new PassManagerImpl(lgcContext->getTargetMachine(), lgcContext->getContext()));
```

Comment:
Nit: It's ususally perferred to use `std::make_unique` instead of plain `new`.
There are also benefits to exception safety, which don't apply for us.

I recall having read a comment elsewhere that you didn't use `std::make_unique` for compile time reasons. Does this also apply here and elsewhere?

Reply:
I don't know; that comment was in code copied from LLVM pass manager code, so I believed it. :-) I guess there was some heavy templating going on in that code.

Reply:
Thanks for the update!